src/go/k8s/tests/e2e/centralized-configuration/03-assert.yaml

```diff
@@ -4,10 +4,6 @@ metadata:
   name: centralized-configuration
 status:
   restarting: true
-  conditions:
-    - type: ClusterConfigured
-      status: "False"
-
```

Comment:
Can you help me understand why this is changed? 

In this test case at point `03` Redpanda is reconfigured (node configuration) that requires redpanda restart. I would assume we need to report that the restart is triggered by configuration rather than changing the container image.

Next test is doing correct test where restarting is `false` and ClusterConfigured set to `true`. Maybe we should drop test 03?

Reply:
So, I was thinking this may need a fix.. The thing is that for node property changes, the `statefulset_update.go` handler intercepts the change and triggers a restart of the cluster, sometimes without letting the configuration controller to set the ClusterConfigured condition. So, it may never be the case that we have both `restarting=true` and `ClusterConfigured=false`, while `restarting=true` that should be kept for enough time for Kuttl to proceed with next step

Reply:
But I still see some errors, so there might be something else...